scripts/dev/create_package.sh

```diff
@@ -630,6 +631,11 @@ function createPackage {
     rm CONTRIBUTING.md
 
     #copy empty example
+    cd $pkg_ofroot
+
+    cd examples
+    find . -type f -name '*.dll' -delete
```

Comment:
dont need to deploy dlls from addons all into place in examples for release download, projectGenerator 

need to test to make sure addon linked dlls are copied at sln build